DESCRIPTION.rst

```diff
@@ -12,12 +12,12 @@ Release Notes
 
 - v2.3.7(December 10,2020)
 
-   - Added support for MFA Cache.
+   - Added support for MFA token caching.
    - Added support for upcoming downscoped GCS credentials.
-   - Tightened pyOpenSSL dependency pin.
-   - Relaxed boto3 dependency pin up to next major release.
-   - Relaxed cffi dependency pin up to next major release.
-   - Async query execution.
+   - Tightened the pyOpenSSL dependency pin.
+   - Relaxed the boto3 dependency pin to include the next major release.
+   - Relaxed the cffi dependency pin to incllude the next major release.
```

Comment:
This is wrong, we pin now up to next major instead of up to next minor releases.
```suggestion
   - Relaxed the boto3 dependency pin up to the next major release.
   - Relaxed the cffi dependency pin up to the next major release.
```